drivers/storage/portworx/rbac.go

```diff
@@ -0,0 +1,284 @@
+package portworx
+
+import (
+	corev1alpha1 "github.com/libopenstorage/operator/pkg/apis/core/v1alpha1"
+	"github.com/portworx/sched-ops/k8s"
+	"k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	pxServiceAccountName     = "px-account"
+	pxServiceName            = "portworx-service"
+	pxSecretsNamespace       = "portworx"
+	pxClusterRoleName        = "node-get-put-list-role"
```

Comment:
at some point, we should rename these and handle upgrades. I had named these wrong 2 years back to start with :) The actual role now has very different permissions.

Reply:
Will add it in a separate PR.